src/app/util/helper.ts

```diff
@@ -17,6 +18,10 @@ export const reduceAddress = (address: string | undefined, lengthLeft: number, l
   return t("wallet.notConnected");
 };
 
+export const urlTo = (url: string, param: UrlProps) => {
+  return `${url}/${param.id}`;
```

Comment:
This is not good implementation, because you are limited to only one param with id. What will happen if you want to pass two or more props?

Reply:
param is an object of type UrlProps you would just add more fields to it, that way you can pass as many props inside param as you want.

Reply:
Yes, but the point is that u hardcoded only id in the return. That should be dynamic. U see... return `${url}/${param.id}`; What if I pass {name=marko}, u will never see that in the return

Reply:
done